index.html

```diff
@@ -57,5 +49,21 @@ <h1 id="name">
         </ul>
     </div>
 </section>
```

Comment:
- [ ] You will notice there are images at the background, you can download it on Figma as one image and adapt it to your project.

## Figma
![image](https://user-images.githubusercontent.com/86200356/234817593-8848f924-ae0c-4868-b6ea-fde6e9b4f906.png)

## Yours 
![image](https://user-images.githubusercontent.com/86200356/234817722-2ebc6ab2-a681-4a5e-af7f-cb67a9a18667.png)

- [ ] The issue with the hamburger applies to the links also.
![image](https://user-images.githubusercontent.com/86200356/234818531-f4e93fd9-fa43-46db-b223-b0bc61dc33e2.png)

Reply:
I think the issue you point out  is coming from your end because they are showing perfectly fine on end